### PR TITLE
Add cd into prombench directory

### PR DESCRIPTION
**Pull request by nikita-mk**
To make the output of subsequent commands visible in the github actions log.
We just need to pass args now.

Signed-off-by: nikita-mk <nikkikokitkar@gmail.com>

### CODE REVIEW

File: cmd/prombench/clone.sh [hunk 1]
```diff
@@ -6,5 +6,6 @@ git clone $PROMBENCH_REPO $PROMBENCH_DIR
 # copy the prombench binary to the cloned directory
 cp /usr/bin/prombench $PROMBENCH_DIR/
 
+cd $PROMBENCH_DIR
 # execute arguments passed to the image
-$@
+$*
```
Comment (krasi-georgiev) [inline cmd/prombench/clone.sh]: why this change?

Comment (nikita-mk) [reply cmd/prombench/clone.sh]: We tried this for github actions. But both `*` and `@` work.

Comment (krasi-georgiev) [reply cmd/prombench/clone.sh]: So what is the answer to the question why do we need this change?

Comment (nikita-mk) [reply cmd/prombench/clone.sh]: In github actions example, `*` was used to accept the args. So we thought that might be the case that github actions was not executing `make` commands. But adding `cd $PROMBENCH_DIR` resolved it. Will revert it back to `@`.
